### PR TITLE
WIP: Miscellaneous

### PR DESCRIPTION
**Pull request by eyalcohen4**
#57

### CODE REVIEW

File: client/src/app/app.ts [hunk 1]
```diff
@@ -23,4 +23,4 @@ export class App {
 
   constructor() {}
 
-}
+}
```
Comment (ordavidil) [inline client/src/app/app.ts]: New line at the end of file is missing.

File: client/src/app/managing-body-list/managing-body-list.component.ts [hunk 2]
```diff
@@ -28,4 +28,4 @@ export class ManagingBodyListComponent implements OnInit {
   getManagingBodies() {
     this.managingBodyList = this.managingBodyService.getManagingBodies();
   }
-}
+}
```
Comment (ordavidil) [inline client/src/app/managing-body-list/managing-body-list.component.ts]: New line at the end of file is missing.

File: client/src/app/managing-body-list/managing-body-list.component.ts [hunk 3]
```diff
@@ -28,4 +28,5 @@ export class ManagingBodyListComponent implements OnInit {
   getManagingBodies() {
     this.managingBodyList = this.managingBodyService.getManagingBodies();
   }
-}
+  
+}
```
Comment (nirgn975) [inline client/src/app/managing-body-list/managing-body-list.component.ts]: New line is missing at the end of the file.

File: client/src/app/managing-body-list/managing-body-list.component.ts [hunk 4]
```diff
@@ -18,7 +18,7 @@ export class ManagingBodyListComponent implements OnInit {
   errorMessage: String;
 
   constructor(
-    private managingBodyService: ManagingBodyService
+      private managingBodyService: ManagingBodyService
```
Comment (nirgn975) [inline client/src/app/managing-body-list/managing-body-list.component.ts]: Remove the whitespace.

File: client/src/app/managing-body-detail/managing-body-detail.component.ts [hunk 5]
```diff
@@ -1,7 +1,6 @@
 import { Component } from '@angular/core';
 
 @Component({
-  selector: 'op-managing-body-detail',
```
Comment (nirgn975) [inline client/src/app/managing-body-detail/managing-body-detail.component.ts]: Why you delete this line?

Comment (eyalcohen4) [inline client/src/app/managing-body-detail/managing-body-detail.component.ts]: In the issue there is a mission to delete all selector's that not in use,
so i checked for one in the html template's for this component and there isn't any.
Thought that this is maybe a misunderstanding of mine.
revert this change?

Comment (nirgn975) [inline client/src/app/managing-body-detail/managing-body-detail.component.ts]: Yeah revert it.
I did not explain myself well, I meant do delete the empty selectors in each component, like in `app` component, we don't use the `pipes` selector (the array is empty -> we don't use it, so delete it.)
